.buildkite/pipeline.yml

```diff
@@ -0,0 +1,73 @@
+# Default pipeline from cli
+
+common_agent: &common_agent
+  agents:
+    queue: ruby-mysql
+    ruby: 2.6.1
+    elasticsearch: 6.3.1
+common_artifact: &common_artifact
+  artifact_paths:
+    - spec/reports/rspec.xml
+common_env: &common_env
+  ELASTICSEARCH_TEST_PORT: '9200'
+common_unit_tests: &common_unit_tests
+  <<: *common_agent
+  <<: *common_artifact
+
+appraisal_commands: &appraisal_commands
+  - bundle install
+  - bundle exec appraisal install
```

Comment:
I know this is already merged, but you can save a good amount of build time by using the `BUNDLE_GEMFILE` environment variable to set the appropriate regression Gemfile. This saves time because then each step just needs to `bundle install` a single environment rather than all environments, and then the commands for all the steps are exactly the same as well with just `bundle install && bundle exec rspec`